test/escape_test.dart

```diff
@@ -3,7 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'package:matcher/src/util.dart';
-import 'package:test/test.dart';
+import 'package:test/test.dart' show group, test, expect, isTrue;
```

Comment:
Should we not just remove the `matcher` import?

Reply:
I know it seems a bit academi. Trying to make sure we're really getting it from matcher. 🤷

Reply:
Why is that important here?